.gitlab-ci.yml

```diff
@@ -0,0 +1,36 @@
+include:
```

Comment:
General question: if we add a `.gitlab-ci.yml` file at the root of this repo, wouldn't that complicate future OSS contributions? Wondering if you had some sort of mitigation strategy for that.